examples/aio-sandbox/README.md

```diff
@@ -0,0 +1,88 @@
+# All-in-One (AIO) Sandbox Example
+
+This example demonstrates how to create and access an [All-in-One (AIO) Sandbox](https://github.com/agent-infra/sandbox) via Agent-Sandbox.
+
+## Create an AIO Sandbox
+
+Apply the sandbox manifest with AIO Sandbox runtime. You can find the latest version [here](https://github.com/agent-infra/sandbox/pkgs/container/sandbox).
+
+```sh
+kubectl apply -f aio-sandbox.yaml
+# sandbox.agents.x-k8s.io/aio-sandbox-example created
+```
+
+They can then check the status of the applied resource.
+Verify sandbox and pod are running:
+
+```sh
+# wait until the sandbox is ready
+kubectl wait --for=condition=Ready sandbox aio-sandbox-example
+
+kubectl get sandbox
+# NAME                  AGE
+# aio-sandbox-example   41s
+kubectl get pod aio-sandbox-example
+# NAME                  READY   STATUS    RESTARTS   AGE
+# aio-sandbox-example   1/1     Running   0          49s
+```
+
+The AIO Sandbox has multiple tools pre-installed, including VNC, VSCode, Jupyter and Terminal in one unified environment.
+
+## Accessing the AIO Sandbox Server
+
+Port forward the aio-sandbox server port.
+
+```sh
+# set ingress host and port
+export INGRESS_HOST="localhost"
+export INGRESS_PORT="8080"
+
+# port forward to the aio-sandbox pod
+kubectl port-forward --address ${INGRESS_HOST} pod/aio-sandbox-example ${INGRESS_PORT}:8080
+```
+
+Setup GATEWAY_URL environment variable:
+
+```sh
+export GATEWAY_URL="http://${INGRESS_HOST}:${INGRESS_PORT}"
+
+echo "$GATEWAY_URL"
+# http://localhost:8080
+```
+
+Connect to the aio-sandbox on a browser via `http://<INGRESS_HOST>:<INGRESS_PORT>`
+
+## Access the AIO Sandbox via Python SDK
+
+```sh
+# set up a virtual environment if needed
+python3 -m venv venv
+source venv/bin/activate
+
+# install the agent-sandbox package
+pip install agent-sandbox==0.0.18
+```
```

Comment:
@lizzzcai Thanks for providing this great example. 

To avoid confusion between the SDKs (because of similar names), could we clarify that `agent-sandbox` SDK is for **controlling the tools inside the AIO image** (like the browser and shell), while [`agentic-sandbox-client`](https://github.com/kubernetes-sigs/agent-sandbox/tree/main/clients/python/agentic-sandbox-client) is the SDK for **provisioning the sandbox infrastructure** (managing the Kubernetes resource lifecycle)?

You could also add a note mentioning that these two SDKs are complementary and can be used together for a complete "infrastructure + application" workflow.